ocr-numbers.md

```diff
@@ -1,36 +1,74 @@
-## Step 1
+# Step One
 
-A simple binary font has been constructed using only pipes and
-underscores.
+To begin with, convert a simple binary font to a string containing 0 or 1.
 
-The number is four rows high, three columns wide:
+The binary font uses pipes and underscores, four rows high and three columns wide.
 
+```
      _   #
     | |  # zero.
     |_|  #
          # the fourth row is always blank
+```
 
+Is converted to "0"
+
+```
          #
       |  # one.
       |  #
          # (blank fourth row)
+```
+
+Is converted to "1"
+
+If the input is the correct size, but not recognizable, your program should return '?'
+
+If the input is the incorrect size, your program should return an error.
+
+# Step Two
 
-Write a program that, given a 3 x 4 grid of pipes, underscores, and
-spaces, can determine whether the the grid represents a zero, a one, or
-garble.
+Update your program to recognize binary strings, replacing garbled numbers with ?
 
-Anything else is considered garble, and can be represented with a '?'
+# Step Three
 
-## Step 2
+Update your program to recognize all numbers 0 through 9, both individually and as part of a larger string.
 
-A simple numeric font has been constructed using only pipes and
-underscores.
+```
+ _ 
+ _|
+|_ 
+   
+```
 
-The number consists of four rows high, three columns wide:
+Is converted to "2"
 
+```
       _  _     _  _  _  _  _  _  #
     | _| _||_||_ |_   ||_||_|| | # decimal numbers.
     ||_  _|  | _||_|  ||_| _||_| #
                                  # fourth line is always blank
+```
+
+Is converted to "1234567890"
+
+# Step Four
+
+Update your program to handle multiple numbers, one per line. When converting several lines, replace the empty lines with commas.
```

Comment:
same thought about empty lines, maybe just say "join the line with commas" or something